crates/lib/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #![deny(clippy::cargo)]
 #![allow(clippy::multiple_crate_versions)] // This should be enforced by cargo-deny
 #![allow(clippy::missing_errors_doc)]
+#![allow(clippy::implicit_hasher)]
```

Comment:
I needed to allow this for my changes, and I noticed we were already allowing elsewhere, so I allowed it globally. By default, [it is an allowed pedantic lint](https://rust-lang.github.io/rust-clippy/master/index.html), and the changes needed to satisfy it aren't worth the squeeze IMO, so I don't feel bad about it, but let me know if you feel otherwise.